src/utility.js

```diff
@@ -11,11 +11,12 @@ const { parse } = require('bcp47');
  */
 function normalizeLanguageCode(langCode) {
   let result = parse(langCode.replace(/[_-]/g, '-'));
-  if (!result || !result.langtag.language || !result.langtag.region) {
+  if (!result || !result.langtag.language) {
     throw new Error(`${langCode} is not a valid language code`);
   }
+  let region = result.langtag.region ? '' : '-' + result.langtag.region.toUpperCase()
```

Comment:
I may be misunderstanding shouldn't this be:
```
let region = result.langtag.region ? ('-' + result.langtag.region.toUpperCase()) : ''
```

Reply:
🤦 that what i get for a last minute refactor